.github/ISSUE_TEMPLATE/questions.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Question/Support/Other
+about: If you have a usage question
+labels: Question
+# This template is based on the scikit-learn template
+# https://github.com/scikit-learn/scikit-learn/blob/8e61534f1087703f476414d8dbd3688282f8eebf/.github/ISSUE_TEMPLATE/usage_question.md
+---
+
+<!--
+**If your issue is a usage question, please submit it in one of these other channels instead:**
+- StackOverflow with the Matplotlib tag: https://stackoverflow.com/questions/tagged/matplotlib
+- The Matplotlib community forum: https://discourse.matplotlib.org/
```

Comment:
this one should come first since it's the one we prioritize.

Reply:
Makes sense. I swapped the order.